eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -1,3 +1,15 @@
+function GetOnboardingFile($docRepoLocation, $moniker) { 
```

Comment:
Is this common? I see something similar in the other PRs but slightly different. Not sure if it is worth standardizing here or not.

Reply:
Not sufficiently. 

.NET: 
`bundlepackages/azure-dotnet.csv`
`bundlepackages/azure-dotnet-preview.csv`
`bundlepackages/azure-dotnet-legacy.csv` 

Java:
`package.json` 

JS:
`ci-configs/packages-latest.json`
`ci-configs/packages-preview.json` 
`ci-configs/packages-legacy.json` 

Python: 
`ci-configs/packages-latest.json`
`ci-configs/packages-preview.json` 
`ci-configs/packages-legacy.json` 

JS and Python could be deduplicated and treated as the "default" implementation if we wanted to do that.